caom2-artifact-resolvers/build.gradle

```diff
@@ -14,7 +14,7 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '1.2.4'
+version = '1.2.6'
```

Comment:
Current published version is 1.2.4 so .5 is sufficient here

Reply:
Right. This is due to the increment in the cadc-9965 branch.